src/app/pages/progress-report/components/progress-report-chart/progress-report-chart.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="chart-wrapper">
+  <div class="empty-chart-text" *ngIf="!isFirstWeekPassed() || isChartEmpty()">
```

Comment:
Can we store the result of `isFirstWeekPassed()` and `isChartEmpty()` in a variable if it won't keep changing since this method will be called repeatedly?